config/betterquesting/DefaultQuests.json

```diff
@@ -8547,21 +8547,21 @@
           "lockedProgress:1": 0,
           "autoClaim:1": 0,
           "isSilent:1": 0,
-          "desc:8": "Because we\u0027re nice developers, we decided to add a little bonus if you happen to compress charcoal. Not only does it allow you to process more than 16 steel at a time (not in the bricked blast furnace), it also yields 10 times the burn time of a piece of charcoal. So one charcoal for free. This bonus only occurs on the first compression step. (But for fanciness, you should definitely get a quintuple one!)"
+          "desc:8": "Because we\u0027re nice developers, we decided to add a little bonus if you happen to compress charcoal. Use it in the bricked blast furnace, or in a boiler, it yields 10 times the burn time of a piece of charcoal. So one charcoal for free. This bonus only occurs on the first compression step. (But for fanciness, you should definitely get a quintuple one!)\n\n§3Of course, if you\u0027re not using BBFs or boilers, they\u0027re kinda useless."
         }
       },
       "tasks:9": {
         "0:10": {
           "partialMatch:1": 1,
           "autoConsume:1": 0,
           "groupDetect:1": 0,
-          "ignoreNBT:1": 0,
+          "ignoreNBT:1": 1,
           "index:3": 0,
           "consume:1": 0,
           "requiredItems:9": {
             "0:10": {
-              "id:8": "dreamcraft:tile.QuintupleCompressedCharcoal",
-              "Count:3": 1,
+              "id:8": "dreamcraft:tile.Charcoal",
+              "Count:3": 64,
```

Comment:
Please change this to the gregtech version that is the recipe output from a compressor.
```json
              "id:8": "gregtech:gt.blockgem3:4",
              "Count:3": 64,
```

Reply:
Done